pyproject.toml

```diff
@@ -16,6 +16,9 @@ exclude = [
     "cognite/neat/explorer-ui/neat-app/node_modules/**",
     "cognite/neat/explorer-ui/neat-app/*.json",
 ]
+[tool.poetry.group.dev.dependencies]
+mkdocs-glightbox = "^0.3.4"
```

Comment:
Needs to be part of the docs dependencies

Reply:
Me need either this plugin or similar that allows zoom/enlarge images